tools/releaseBuild/azureDevOps/WindowsBuild.yml

```diff
@@ -0,0 +1,507 @@
+    displayName: 'Remove all containers'
+    continueOnError: true
+
+  - powershell: |
+       docker image ls --format '{{ json .}}'|ConvertFrom-Json|% {
+           if($_.tag -eq '<none>')
+           {
+               $formatString = 'yyyy-MM-dd HH:mm:ss zz00'
+               $createdAtString = $_.CreatedAt.substring(0,$_.CreatedAt.Length -4)
+               $createdAt = [DateTime]::ParseExact($createdAtString, $formatString,[System.Globalization.CultureInfo]::InvariantCulture)
+               if($createdAt -lt (Get-Date).adddays(-1))
+               {
+                    docker image rm $_.ID
+               }
+           }
+       }
+       exit 0
+    displayName: 'Remove old images'
+    continueOnError: true
+
+  - powershell: |
+       Write-verbose "--docker info---" -verbose
+       docker info
+       Write-verbose "--docker image ls---" -verbose
+       docker image ls
+       Write-verbose "--docker container ls --all---" -verbose
+       docker container ls --all
+       Write-verbose "--git branch ---" -verbose
+       git branch
+       exit 0
+    displayName: 'Get Environment'
+
+  - powershell: |
+       ./tools/releaseBuild/vstsbuild.ps1 -ReleaseTag $(ReleaseTag) -Name win-x64-component-registration
+    displayName: 'Build Windows Universal - Component Registration'
+
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+    displayName: 'Component Detection'
+    inputs:
+      sourceScanPath: '$(componentregistration)'
+      snapshotForceEnabled: true
+
+- job: CreateAzureContainerJob
+  displayName: Create Azure container and set access
+  dependsOn: BuildJob
+  condition: succeeded()
+  pool:
+    name: Hosted VS2017
+
+  steps:
+
+  - powershell: |
+      ./tools/releaseBuild/setReleaseTag.ps1 -ReleaseTag $(ReleaseTag)
+    displayName: 'Set ReleaseTag Variable'
+
+  - powershell: |
+       $azureVersion = $env:ReleaseTag.ToLowerInvariant() -replace '\.', '-'
+       $vstsCommandString = "vso[task.setvariable variable=AzureVersion]$azureVersion"
+       Write-Host "sending " + $vstsCommandString
+       Write-Host "##$vstsCommandString"
+    displayName: 'Set ContainerName'
+
+  - task: AzureCLI@1
+    displayName: 'Create container'
+    inputs:
+      azureSubscription: '$(AzureFileCopySubscription)'
+      scriptLocation: inlineScript
+      inlineScript: |
+       az storage container create --account-name $(StorageAccount) --account-key $(StorageAccountKey) --name $(AzureVersion) --public-access blob
+
+  - task: AzureCLI@1
+    displayName: 'Set permissions on container'
+    inputs:
+      azureSubscription: '$(AzureFileCopySubscription)'
+      scriptLocation: inlineScript
+      inlineScript: 'az storage container set-permission --account-name $(StorageAccount) --account-key $(StorageAccountKey) --name $(AzureVersion) --public-access blob'
```

Comment:
remove this step, we should be able to eliminate it.

Reply:
Removed.